internal/k8s/controller.go

```diff
@@ -294,13 +326,18 @@ func AddRouteEventHandler(numWorkers uint32, c *AviController) cache.ResourceEve
 			newRoute := cur.(*routev1.Route)
 			if isRouteUpdated(oldRoute, newRoute) {
 				namespace, _, _ := cache.SplitMetaNamespaceKey(utils.ObjKey(newRoute))
-				key := utils.OshiftRoute + "/" + utils.ObjKey(newRoute)
-				bkt := utils.Bkt(namespace, numWorkers)
-				if !lib.HasValidBackends(newRoute.Spec, newRoute.Name, namespace, key) {
-					status.UpdateRouteStatusWithErrMsg(newRoute.Name, namespace, lib.DuplicateBackends)
+				nsFilterObj := utils.GetGlobalNSFilter()
+				if utils.CheckIfNamespaceAccepted(namespace, nsFilterObj, nil, true) {
```

Comment:
Nit: Can you move the return logic and remove the else?


```
if !utils.CheckIfNamespaceAccepted(namespace, nsFilterObj, nil, true) {
// log and return
}
// enqueue to work queue. 
```
https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow